singularity-eos/base/eos_error.hpp

```diff
@@ -0,0 +1,26 @@
+//------------------------------------------------------------------------------
+// © 2021. Triad National Security, LLC. All rights reserved.  This
+// program was produced under U.S. Government contract 89233218CNA000001
+// for Los Alamos National Laboratory (LANL), which is operated by Triad
+// National Security, LLC for the U.S.  Department of Energy/National
+// Nuclear Security Administration. All rights in the program are
+// reserved by Triad National Security, LLC, and the U.S. Department of
+// Energy/National Nuclear Security Administration. The Government is
+// granted for itself and others acting on its behalf a nonexclusive,
+// paid-up, irrevocable worldwide license in this material to reproduce,
+// prepare derivative works, distribute copies to the public, perform
+// publicly and display publicly, and to permit others to do so.
+//------------------------------------------------------------------------------
+
+#ifndef SINGULARITY_EOS_BASE_EOS_ERROR_HPP_
+#define SINGULARITY_EOS_BASE_EOS_ERROR_HPP_
```

Comment:
This appears to follow a convention of `SINGULARITY_EOS_<FILEPATH>_<HEADERNAME>`. I think we should make this consistent across all of our header files (mixed_cell_models.hpp and eos_builder.hpp are examples where this is not consistently applied).

Reply:
Yes, I agree. I'll try to go through and check these.

Reply:
I think I've fixed this everywhere. I added a leading underscore too.